contrib/metacontroller/README.md

```diff
@@ -0,0 +1,9 @@
+# Metacontroller
+
+- [Official documentation](https://metacontroller.github.io/metacontroller/)
+- [Official repoitory](https://github.com/metacontroller/metacontroller)
+
+## Upgrade
+
+To update this component pull the latest content from [Kubeflow Pipelines third-party folder](https://github.com/kubeflow/pipelines/tree/master/manifests/kustomize/third-party/metacontroller).
```

Comment:
Can we use an environment variable `KFP_VERSION` to make sure we fetch the correct version of metacontroller?

For example:
```
export KFP_VERSION=2.0.0-alpha.3
kpt pkg get "https://github.com/kubeflow/pipelines/tree/${KFP_VERSION}/manifests/kustomize/third-party/metacontroller" ./contrib/metacontroller
```

This is just an example script which I haven't validated, we can also just ask people to pull from a tag manually.

Reply:
This is a great idea. Thank you!

Reply:
Thank you for the update!